Foundation/JSONEncoder.swift

```diff
@@ -861,7 +904,7 @@ open class JSONDecoder {
     open func decode<T : Decodable>(_ type: T.Type, from data: Data) throws -> T {
```

Comment:
This function looks very different to the overlay - https://github.com/apple/swift/blob/master/stdlib/public/SDK/Foundation/JSONEncoder.swift#L1102 ??

Reply:
I have applied overlay changes on top of #1347. However still this implementation depends on ``` bridgeToObjectiveC``` instead of ```ConevertedKey``` method.Hence only the changes pertaining to the ```ConvertedKey``` have been ignored.